sources/HUBViewControllerImplementation.m

```diff
@@ -956,14 +958,15 @@ - (void)loadImageFromData:(id<HUBComponentImageData>)imageData
     if (CGSizeEqualToSize(preferredSize, CGSizeZero)) {
         return;
     }
-    
+
     HUBComponentImageLoadingContext * const context = [[HUBComponentImageLoadingContext alloc] initWithImageType:imageData.type
                                                                                                  imageIdentifier:imageData.identifier
                                                                                                wrapperIdentifier:componentWrapper.identifier
-                                                                                                      childIndex:childIndex];
+                                                                                                      childIndex:childIndex
+                                                                                                       timestamp:CACurrentMediaTime()];
```

Comment:
It doesn't sync with the server, as in the network right? I think it's fine to rely on `NSDate`.